blocks/library/list/index.js

```diff
@@ -7,6 +7,34 @@ import Editable from 'components/editable';
 
 const { children, prop } = hpq;
 
+function activeEditorExecCommand( command ) {
+	return () => {
+		const ed = tinymce.activeEditor;
```

Comment:
I managed to avoid using the global but it required passing the editor via a callback.